deltachat-jsonrpc/src/api.rs

```diff
@@ -1530,13 +1530,14 @@ impl CommandApi {
 
     /// Resets contact encryption.
     async fn reset_contact_encryption(&self, account_id: u32, contact_id: u32) -> Result<()> {
-        let ctx = self.get_context(account_id).await?;
         let contact_id = ContactId::new(contact_id);
+        let ctx = self.get_context(account_id).await?;
 
         contact_id.reset_encryption(&ctx).await?;
         Ok(())
     }
 
+    /// Sets display name for existing contact.
     async fn change_contact_name(
```

Comment:
It's not clear how to use this API to reset the display name to the auth name (given by the contact itself). Probably by passing an empty string, this should be documented or `Option<String>` passed instead